data/mods/Magiclysm/snippets/monster_parrots.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "type": "speech",
+    "speaker": [ "mon_mi_go_fleshwarper", "mon_mi_go_nethermancer" ],
+    "sound": "\"Abra-cadabra!\"",
+    "volume": 20
+  },
+  {
+    "type": "speech",
+    "//~": "The mi-go is reciting a Latin incantation here that should not be translated.",
+    "speaker": [ "mon_mi_go_fleshwarper", "mon_mi_go_nethermancer" ],
+    "sound": "\"Per Adonai Elohim, Adonai Jehova, Adonai Sabaoth…\"",
```

Comment:
Why does the mi-go recite Latin, a language it likely wouldn’t know? Latin hasn’t been spoken for a very long time, and few people in New England would know it enough to speak, much less for a mi-go to parrot.

Reply:
Magiclysm never really defines the way spellcasting occurs, but I was assuming at least some people do it through "esoteric" languages, so there would be more people who spoke at least some Latin, Sanskrit, Ancient Greek, Biblical Hebrew, Classical Chinese, and so on than in our world.

Reply:
I feel like the Mi-go wouldn't use these "esoteric" languages of our universe, but one that comes from their own. This might involve having specialized limbs that pulse in certain ways to put their mind in a trance-like state that they use for spellcasting and focusing magical energy, as they wouldn't have time here on Earth to learn these native languages and speak them in the capacity required for spellcasting. It'd also fit more with the alien theme of them, as they are from beyond this world and don't have many ties to it. This would also reflect their unique culture and societal outlook; biology is preferred for magic and technology over conventional engineering practices.

Reply:
I agree that makes more sense for how the mi-go cast their own spells, but that wasn't the purpose of the Latin I included. They aren't use it to cast their own spells. Rather, I put it in as a parrot, the way the other mi-go also parrot things, since I figured that the mi-go would send mages to capture mages (fighting fire with fire, etc) and therefore the migo's own wizards would be more likely to parrot mystical phrases they've heard in the course of their raids. 

(And, from a design perspective, so these mi-go have unique parrots to warn the player they're nearby since they're much more dangerous than a normal mi-go)

Reply:
Okay, this makes a lot more sense now. I'd recommend including that in the code notes so nobody gets confused on it in the future.